### DISCUSSION TITLE
How can I use <Card /> component in my docs-theme when deploying on Vercel?

### DISCUSSION THREAD

**Discussion by bogdanmosica** [shuding/nextra#1656] [Q&A]
How can I use <Card /> component in my docs-theme when deploying on Vercel? This a component I saw it is used on official documentation of Nextra.

For some reason, on my local dev mode, the <Card> Component seems to be working fine, but when I try to deploy on Vercel, I get the following error: 

<html>
<body>

./pages/introduction.mdx
--
15:57:06.102 | Attempted import error: 'Card' is not exported from 'nextra-theme-docs' (imported as 'Card').
15:57:06.102 |  
15:57:06.102 | ./pages/introduction.mdx
15:57:06.102 | Attempted import error: 'Cards' is not exported from 'nextra-theme-docs' (imported as 'Cards').

</body>
</html>

I have tried to search on google but nothing shows up. I would guess this component is used only for the official project?

**Comment by dimaMachina** [comment]
Try to update nextra, it was exported in the newly versions

**Reply by bogdanmosica** [reply]
> Try to update nextra, it was exported in the newly versions

I npm installed today and in my package.JSON I have latest for nextra.
